### DISCUSSION TITLE
Secured connection

### DISCUSSION THREAD

**Discussion by oleksandrmelnychenko** [microsoft/msquic#1747] [Q&A] [answered]
What type of data should be used in the server configuration for secured connection? 

![image](https://user-images.githubusercontent.com/7407215/122636436-ee619380-d0f1-11eb-84f3-7ba96ff3fa5e.png)

**Comment by nibanks** [comment, marked as answer]
It looks like your `cert` and `privateKey` variables are the contents of the files, not the file names/paths themselves? `CertFile.CertificateFile` and `CertFile.PrivateKeyFile` are supposed to be set to the file names/paths. We don't currently support passing contents directly (though I suspect it would be easy enough to do).
